R/contingencytables.R

```diff
@@ -409,13 +412,13 @@ ContingencyTables <- function(jaspResults, dataset, options, ...) {
   if (options$VovkSellkeMPR)  table$addColumnInfo(name = paste0("MPR[",   fold, "]"),   title = gettextf("VS-MPR%s", "\u002A"), type = "number")
 }
 
-.crossTabLogOddsAddColInfo <- function(table, fold, ci.label) {
+.crossTabOddsAddColInfo <- function(table, fold, ci.label) {
   table$addColumnInfo(name = paste0("type[",  fold, "]"), title = "",                        type = "string")
-  table$addColumnInfo(name = paste0("value[", fold, "]"), title = gettext("Log Odds Ratio"), type = "number")
+  table$addColumnInfo(name = paste0("value[", fold, "]"), title = gettext("Odds Ratio"),     type = "number")
```

Comment:
```suggestion
if (options[["LogOdds"]] {
  table$addColumnInfo(name = paste0("value[", fold, "]"), title = gettext("Odds Ratio"),     type = "number")
} else {
  table$addColumnInfo(name = paste0("value[", fold, "]"), title = gettext("Log Odds Ratio"),     type = "number")
}
```
it makes some sense to show log in the title, if the value is also on the log scale.